arkouda/pdarrayclass.py

```diff
@@ -66,6 +66,41 @@ def unescape(s):
         raise ValueError(("unsupported value from server {} {}".\
                               format(mydtype.name, value)))
 
+
+@typechecked
+def _parse_single_int_array_value(msg : str) -> object:
+    """
+    Attempt to convert a scalar return value from the arkouda server to a
+    numpy string in Python. The user should not call this function directly.
+
+    Parameters
+    ----------
+    msg : str
+        scalar value in string form to be converted to a numpy string
+
+    Returns
+    -------
+    object numpy scalar
+    """
+    fields = msg.split(" ", 1)
+    dtname, value = msg.split(maxsplit=1)
+    mydtype = dtype(dtname)
+    try:
+        if mydtype == akint64:
+            nfields = value.split("\"")
+            # original we return a string include the last ending 0
+
+            _, sastr = nfields[1].split(maxsplit=1)
+            tmpstr = sastr.split()
+            intary = [int(numeric_string) for numeric_string in tmpstr]
+            return intary
+            # now we return a suffix array and not include the last ending 0
+        else:
+            raise ValueError(("not correct int data type from server {} {}".format(mydtype.name, value)))
+    except:
+        raise ValueError(("unsupported value from server {} {}".format(mydtype.name, value)))
```

Comment:
I recommend passing in the Exception as e into ValueError to get more context for debug purposes